src/histolab/slide.py

```diff
@@ -173,8 +173,7 @@ def thumbnail_path(self) -> str:
 
     # ---private interface methods and properties---
 
-    @staticmethod
```

Comment:
Why removing `@staticmethod`? The method is not accessing the instance nor the class, and it's independent from everything else around it.

Reply:
Sure thing, but unittes+mcok has a known issue when we use staticmethods decorators, when we figure out how to fix this kind of problems we can re-decorate that methods

Reply:
Didn't know about that!